### ISSUE TITLE
[css-images] RTL issue in radial-gradient() and conic-gradient()

### ISSUE THREAD

**Issue by ramiy** [w3c/csswg-drafts#6270]
Both [Radial Gradients](https://drafts.csswg.org/css-images-4/#radial-gradients) and [Conic Gradients](https://drafts.csswg.org/css-images-4/#conic-gradients) are **not RTL compatible**. When using RTL languages, we need the reverse everything.

Currently both `radial-gradient()` and `conic-gradient()` support only one direction, **clockwise**. There is not support for **counter-clockwise**.

There should be a way do set counter-clockwise direction.

### Possible solution

In redial-gradient we have keywords like `closest-side`, `closest-corner`, `farthest-side` and `farthest-corner`.

There should be two "direction" keywords `clockwise` and `counter-clockwise`. Why two? do override previously defined settings.

**Comment by ramiy** [comment]
@LeaVerou please share your thoughts on this issue.

**Comment by LeaVerou** [comment]
Yeah, we definitely need logical keywords in addition to the physical ones already there.

I'm confused about the `clockwise` and `counter-clockwise` proposal. Is that a separate proposal or does it somehow help with bidi?

**Comment by ramiy** [comment]
To clarify my solution, I was talking about something like this:

```css
background: conic-gradient(at 50% 50% clockwise, #f06, gold);
background: conic-gradient(at 50% 50% counter-clockwise, #f06, gold);

background: conic-gradient(from 45deg clockwise, white, black, white);
background: conic-gradient(from 45deg counter-clockwise, white, black, white);
```

You can use other keywords or propose other solutions.

As long as I'll able to reverse everything, I don't care about the "how".
